SygicMapsKit/Classes/NavigationModule/SYMKNavigationViewController.swift

```diff
@@ -183,6 +183,15 @@ public class SYMKNavigationViewController: SYMKModuleViewController {
         if let instructionsController = instructionsController {
             navigationView.setupInstructionView(instructionsController.view)
         }
+        if useInfobar {
+            setupInfobarController()
+            if let infobarController = infobarController {
+                navigationView.setupInfobarView(infobarController.infobarView)
+            }
+        }
+        if useSpeedControls {
```

Comment:
nechceli sme to rozdelit aby si mohol mat samostatne speedlimit bez current speedu?

Reply:
Aha, neviem po pravde, opytam sa Tomasa ako to maju oni.

Reply:
Jo, maju to ako 2 samostatne kontrolky v podstate, takze to prerobim tiez tak.